### DISCUSSION TITLE
Important values missing

### DISCUSSION THREAD

**Discussion by DeusHyper** [kutu/pyirsdk#44] [General]
Hey,

first up, thank you for this great library!
I would just like to know if it was possible to add values like hybrid regeneration / deploy or battery status since they are very important values if you drive a gtp. I am currently building my own dashboard since I have much more experience with python than with SimHub, however SimHub can read out these values.

Kind regards,
Jonas

**Comment by kutu** [comment]
https://github.com/kutu/pyirsdk/issues/33#issuecomment-1711262491

**Reply by DeusHyper** [reply]
Perfect, thanks you!
